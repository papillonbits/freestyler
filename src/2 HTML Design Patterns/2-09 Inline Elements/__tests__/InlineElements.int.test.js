import { regular } from './InlineElements.int.story'

describe('<InlineElements />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
