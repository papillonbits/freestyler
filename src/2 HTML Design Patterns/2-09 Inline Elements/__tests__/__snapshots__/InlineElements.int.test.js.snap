// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InlineElements /> Render must match regular() 1`] = `
<div>
  <div
    class="inline-elements"
  >
    <h1>
      Inline Elements
    </h1>
    <h2>
      Italicized
    </h2>
    <code>
      &lt;em&gt; 
    </code>
     
    <em>
      emphasized
    </em>
     
    <br />
    <code>
      &lt;cite&gt; 
    </code>
     
    <cite>
      citation
    </cite>
     
    <br />
    <code>
      &lt;var&gt; 
    </code>
     
    <var>
      computer variable
    </var>
     
    <br />
    <code>
      &lt;dfn&gt; 
    </code>
     
    <dfn>
      definition
    </dfn>
     
    <br />
    <h2>
      Bold
    </h2>
    <code>
      &lt;strong&gt; 
    </code>
     
    <strong>
      strongly emphasized
    </strong>
     
    <br />
    <h2>
      Monospace
    </h2>
    <code>
      &lt;code&gt; 
    </code>
     
    <code>
      computer code
    </code>
     
    <br />
    <code>
      &lt;kbd&gt; 
    </code>
     
    <kbd>
      key press
    </kbd>
     
    <br />
    <code>
      &lt;samp&gt; 
    </code>
     
    <samp>
      sample computer output
    </samp>
     
    <br />
    <h2>
      Underlined
    </h2>
    <code>
      &lt;a&gt; 
    </code>
     
    <a
      href="#"
    >
      a
    </a>
     
    <br />
    <code>
      &lt;acronym&gt; 
    </code>
     
    <acronym
      title="a"
    >
      acronym
    </acronym>
     
    <br />
    <code>
      &lt;abbr&gt; 
    </code>
     
    <abbr
      title="a"
    >
      abbreviation
    </abbr>
     
    <br />
    <h2>
      Vertical-aligned
    </h2>
    <code>
      &lt;sup&gt; 
    </code>
     superscript
    <sup>
      1
    </sup>
     
    <br />
    <code>
      &lt;sub&gt; 
    </code>
     subscript
    <sub>
      1
    </sub>
     
    <br />
  </div>
</div>
`;
